### DISCUSSION TITLE
Uploading screenshots not working

### DISCUSSION THREAD

**Discussion by praeclarum** [community/community#49811] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Hi, usually I can drag a file into the Markdown editor. When I've been trying to do that today, the "Uploading" text appears briefly and then disappears. No error is show, no image appears.

Safari for Mac v16.2

**Comment by bbkx226** [comment, marked as answer]
It's possible that there may be a temporary issue with the Markdown editor or with the browser itself. Here are a few things you can try to troubleshoot the issue:

1. Clear your browser cache and cookies: Sometimes, clearing your browser's cache and cookies can help fix issues like this. To do this in Safari, go to Safari > Preferences > Privacy, then click "Manage Website Data" and select "Remove All" to clear your cache and cookies.

2. Try using a different browser: If clearing your cache and cookies doesn't work, try using a different browser (e.g. Chrome, Firefox) to see if the issue persists. If it doesn't, it may be an issue with Safari specifically.

3. Check your file size: Make sure that the file you are trying to upload is not too large. Most Markdown editors have a limit on the size of files that can be uploaded.

**Comment by praeclarum** [comment]
Embarrassingly, I did miss the error saying **the file was too big**. The error is shown at the bottom of the editor which is off screen so I didn't see it. Perhaps an alert would be a better error delivery UI when dropping files.
